client/src/modules/Mentor/pages/InterviewWaitingList/index.tsx

```diff
@@ -135,6 +140,22 @@ export function InterviewWaitingList({ session, course, interview }: PageProps)
                 >
                   <Button type="link">Want to interview</Button>
                 </Popconfirm>
+                {isStageInterview && isMentor(session, courseId) ? (
+                  <Popconfirm
+                    title={
+                      <>
+                        Are you sure you want to assign <b>{record.githubId}</b> to yourself without an interview?
+                      </>
+                    }
+                    okText="Yes"
+                    okButtonProps={{ disabled: !record.rating }}
+                    onConfirm={() => assignStudentToMentor(record.githubId)}
+                  >
+                    <Button disabled={!record.rating} type="link">
```

Comment:
curious what is the purpose of this disable, if there is no rating, maybe better to hide button or in a tooltip explain why it is disabled